src/lib/utils/Orientation.js

```diff
@@ -18,10 +18,18 @@ export const getOriginalScreenHeight = () => originalScreenHeight
 export const getOriginalScreenWidth = () => originalScreenWidth
 
 export const getScreenHeight = () => Dimensions.get('window').height
-export const getMaxDeviceHeight = () => (isMobileOnly ? getScreenHeight() : theme.sizes.maxHeightForTabletAndDesktop)
+export const getMaxDeviceHeight = () => {
+  const height = getScreenHeight()
+
+  return Math.min(height, theme.sizes.maxHeightForTabletAndDesktop)
+}
 
 export const getScreenWidth = () => Dimensions.get('window').width
-export const getMaxDeviceWidth = () => (isMobileOnly ? getScreenWidth() : theme.sizes.maxWidthForTabletAndDesktop)
+export const getMaxDeviceWidth = () => {
+  const width = getScreenWidth()
+
+  return Math.min(width, theme.sizes.maxWidthForTabletAndDesktop)
```

Comment:
@serdiukov-o-nordwhale incorrect logic for max width/height
on mobile it should always take the whole device widht/height

Reply:
```javascript
export const getMaxDeviceWidth = () => {
  const width = getScreenWidth()
  const { maxWidthForTabletAndDesktop }  = theme.sizes

 return isMobileOnly ? width : Math.min(width, maxWidthForTabletAndDesktop)
}
```
  
?